src/core/src/package_managers/AzL3TdnfPackageManager.py

```diff
@@ -51,14 +48,11 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
             execution_config.included_classifications_list = [Constants.PackageClassification.CRITICAL, Constants.PackageClassification.SECURITY, Constants.PackageClassification.OTHER]
 
     # region Strict SDP using SnapshotTime
-    @staticmethod
-    def __generate_command_with_snapshotposixtime_if_specified(command_template, snapshot_posix_time=str()):
-        # type: (str, str) -> str
-        if snapshot_posix_time == str():
-            return command_template.replace('<SNAPSHOTTIME>', str())
+    def _Base__add_additional_parameters_as_required_to_cmd(self, cmd):
```

Comment:
Overriding this in the base would change the behaviors of the functions calling it right?

Which was the defining ask in that the parent class behavior would effectively morph to what we need just by this override. Or behavioral override.

To the question of what you have in line 66, the property would not be settable in the parent class and it would effectively blank out the suffix (exactly as you've defined this command).

Will discuss over a call to close this as this nature of change (light-weight, but nuanced) is more quickly covered that way..